.github/workflows/test.yml

```diff
@@ -1,12 +1,10 @@
 name: test
 on:
   push:
-    branches:
-    - "main"
```

Comment:
I suggest keeping this branch filter. It'll already run on pull requests, and otherwise it'll run double on pull requests

Reply:
Ok，will revert it